src/mongo/db/commands/dbcommands.cpp

```diff
@@ -371,7 +372,7 @@ class CmdProfile : public ErrmsgCommandDeprecated {
 
         const BSONElement slow = cmdObj["slowms"];
         if (slow.isNumber()) {
-            serverGlobalParams.slowMS = slow.numberInt();
+            uassertStatusOK(db->setSlowMS(slow.numberInt()));
```

Comment:
How does an operator change the _global_ slowMS? We closed [SERVER-5266](https://jira.mongodb.org/browse/SERVER-5266) as "Works as designed", but maybe that could be revived as an approach.

We should also decide if it's OK to change the behavior of this command for existing users, or if we'll need to add new syntax or a new command for setting per-db slowMS values. @asya999 and I will try to spend some time on this question and get back to you.

Reply:
the global slowMS will not be changed at more,  and the value of it is the "default" slowMS for all the database.  I know this request changed the default behavior,  I think you are already thinking if the behavior should be changed or not. 

please let me know your decision if you have made it since I have another feature called "invisible" index want to send...